src/main/java/seedu/duke/command/ListIngredientCommand.java

```diff
@@ -9,6 +9,6 @@
 public class ListIngredientCommand extends Command{
```

Comment:
need to add space after Command
```suggestion
public class ListIngredientCommand extends Command {
```